src/licensedcode/plugin_license.py

```diff
@@ -116,3 +119,57 @@ def get_scanner(
             license_text_diagnostics=license_text_diagnostics,
             license_url_template=license_url_template
         )
+
+    def process_codebase(self, codebase, **kwargs):
+        
+        if codebase.has_single_resource:
+            return
+
+        for resource in codebase.walk(topdown=False):
+            match_reference_license(resource,codebase)
+
+
+def match_reference_license(resource, codebase):
+    """
+    Find instances for any licenses in referenced filenames
+    """
+    if not resource.is_file:
+        return
+
+    licenses = resource.licenses
+    license_expressions = resource.license_expressions
+    if not licenses:
+        return 
+
+    reference_licenses=[]
```

Comment:
what about rather thsi name? also please use proper spacing around = here and in many other places
```suggestion
    referenced_licenses = []
```